Makefile.am

```diff
@@ -50,6 +50,7 @@ pkgbackend_PROGRAMS = parallel serial beh implicitclass
 parallel_SOURCES = \
 	backend/parallel.c
 parallel_LDADD = \
+	libfontembed.la \
```

Comment:
Make sure that this line does not get added back in, it is wrong and therefore I have removed it in commit 8e91b0b572 and did the correct thing to trigger the build of libfontembed via commit d33583e01,

Reply:
Updated PR